CODE_OF_CONDUCT.md

```diff
@@ -2,7 +2,7 @@
 
 As contributors and maintainers of this project, we pledge to respect all people who contribute through reporting issues, posting feature requests, updating documentation, submitting pull requests or patches, and other activities.
 
-We are committed to making participation in this project a harassment-free experience for everyone, regardless of level of experience, gender, gender identity and expression, sexual orientation, disability, personal appearance, body size, race, age, or religion.
+We are committed to making participation in this project a harassment-free experience for everyone, regardless of level of experience, gender, gender identity and expression, sexual orientation, disability, personal appearance, body size, race, ethnicity, age, or religion.
```

Comment:
To me, "political affiliation" and "political ideology" are related, but not the same thing.

"Political affiliation" means what political groups you're affiliated with, whereas "political ideology" refers to what specific political beliefs you yourself possess. I don't think discrimination based on either of those aspects is acceptable.

On the other hand though, trying to create a list of every conceivable thing people could experience prejudice for is probably a pointless endeavor, so maybe just picking one or the other would work.